RELEASE.md

```diff
@@ -0,0 +1,21 @@
+# Releasing `matrix-rust-sdk`
```

Comment:
This document seems to be specifically about breaking-change releases. I think patch releases should generally not require publishing matching versions of all the other crates. Would be nice to note that.

Reply:
Yeah, I suppose it's a discussion to have in the open: to indicate compatibility of crates, we've decided that we'd be using a common minor (until after we get after >= 1.0, then we'd use major) version number for all the crates. I suppose we don't have to do it with patch versions, even though it's a bit breaking symmetry with the other numbers.

Reply:
I'll leave it like this, and we can further discuss what we do with respect to patch versions, as it has implications on the tags too. Should we push a tag for a patch version? But if so, we'd need to tag each crate individually, which is inefficient and not so useful — and we can't override the previous tag without causing trouble. Do we even want to do patch releases? They would need to live on separate branches, and that easily becomes quite messy...